src/Services/MediaService.php

```diff
@@ -138,4 +154,21 @@ protected function createTempDir(string $name): void
             );
         }
     }
+
+    protected function createPreviews(string $fileName, array $previewTypes = null): array
+    {
+        if (is_null($previewTypes)) {
+            $previewTypes = $this->previewDrivers;
+        }
+
+        $results = [];
+
+        foreach ($previewTypes as $type) {
+            $results[$type] = match ($type) {
+                'file' => $this->createPreview($fileName),

```

Comment:
Let's rename method.

```suggestion
                'file' => $this->createFilePreview($fileName),

```